platform/platform_tests/local_country_file_tests.cpp

```diff
@@ -112,25 +114,29 @@ UNIT_TEST(LocalCountryFile_DiskFiles)
 
     string const mapFileName = GetFileName(countryFile.GetName(), MapOptions::Map,
                                            version::FOR_TESTING_TWO_COMPONENT_MWM1);
-    ScopedFile testMapFile(mapFileName, ScopedFile::Mode::Create);
+
+    string const mapFileContents("map");
+    ScopedFile testMapFile(mapFileName, mapFileContents);
 
     localFile.SyncWithDisk();
     TEST(localFile.OnDisk(MapOptions::Map), ());
     TEST(!localFile.OnDisk(MapOptions::CarRouting), ());
     TEST(!localFile.OnDisk(MapOptions::MapWithCarRouting), ());
-    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+    TEST_EQUAL(mapFileContents.size(), localFile.GetSize(MapOptions::Map), ("Size of the content (word \"map\") should be 3."));
```

Comment:
drop the message, please, it is redundant now

Reply:
Done. And I've merge review fix commits.